### PR TITLE
Update deployment

### PR DESCRIPTION
**Pull request by aju**

### CODE REVIEW

**Comment by vergilius** [comment]
:+1:

File: Dockerfile [hunk 1]
```diff
@@ -4,15 +4,12 @@ RUN locale-gen en_US.UTF-8
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
+ENV AWS_ACCESS_KEY ${AWS_ACCESS_KEY}
+ENV AWS_SECRET_ACCESS_KEY ${AWS_SECRET_ACCESS_KEY}
 
 RUN apt-get update
```
Comment (winglot) [inline Dockerfile]: You don't need this.

File: Dockerfile [hunk 2]
```diff
@@ -4,15 +4,12 @@ RUN locale-gen en_US.UTF-8
...
-RUN apt-get install -y openjdk-7-jre-headless
-RUN gem install s3_website --no-rdoc --no-ri
-RUN gem install scss_lint --no-rdoc --no-ri
-RUN gem install jekyll --no-rdoc --no-ri
-
-RUN npm install gulp -g
-RUN npm install http-server -g
+RUN gem install scss_lint jekyll --no-rdoc --no-ri
+RUN npm install gulp http-server -g
```
Comment (winglot) [inline Dockerfile]: One run command is ok here:
```
RUN gem install scss_lint jekyll --no-rdoc --no-ri &&
    npm install gulp http-server -g
```

**Comment by winglot** [comment]
I have one more concern - docker image `shawnzhu/ruby-nodejs` that is used here looks ... well bad, with ubuntu 12.04 and node 0.12. The base image size is 600mb. How about using something modern (like Alpine Linux) with node 5.x?

You can use this as base image https://hub.docker.com/r/mhart/alpine-node/ and install ruby on it.

**Comment by aju** [comment]
@winglot updated

**Comment by winglot** [comment]
:+1:

**Comment by aju** [comment]
This fix requires changes in ansible - updating run command to `gulp deploy` for branches and `gulp deploy -- --production` for master

**Comment by aju** [comment]
OK - tested

**Pull request merged**